### PR TITLE
Fixed more places to disable connecion pool for UDS.

### PR DESCRIPTION
**Pull request by vadim-kovalyov**
Continuation of previous PR #283. Fixed more places to disable connection pool for UDS.

### CODE REVIEW

**Comment by vadim-kovalyov** [comment]
Moving to draft until e2e tests are completed.

**Review by arsing — APPROVED** [review]

**Review by gordonwang0 — APPROVED** [review]

File: .github/workflows/e2e-tests-manual.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
       run: |
         ./ci/e2e-tests.sh ${{ matrix.test_name }}
       env:
-        BRANCH: "${{ github.events.input.branch }}"
```
Comment (arsing) [inline .github/workflows/e2e-tests-manual.yaml:42]: Line 36 too.

Comment (vadim-kovalyov) [reply .github/workflows/e2e-tests-manual.yaml:42]: oh, sorry, fixed that.

File: docs-dev/e2e-tests.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ az role assignment create \
 Next, the identity of this SP and the name of this resource group must be set in GitHub secrets on the repo:
 
 - `AZURE_TENANT_ID`: The `tenant` property from the `az ad sp create-for-rbac` output.
-- `AZURE_USERNAME`: The `AZURE_SP_NAME` variable in the script.
+- `AZURE_USERNAME`: The `appId` property from the `az ad sp create-for-rbac` output.
```
Comment (vadim-kovalyov) [inline docs-dev/e2e-tests.md:45]: Name did not work for me. According to docs, it should be app id instead. https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli#sign-in-using-a-service-principal

Comment (arsing) [reply docs-dev/e2e-tests.md:45]: Name works for me.

```sh
$ az login --service-principal --tenant '<guid>' --username 'http://iot-identity-service-e2e-tests' '--password=<password>'
```

Comment (vadim-kovalyov) [reply docs-dev/e2e-tests.md:45]: Hm, it did not work when I put this name as GH Secret. I made sure I used `http://iot-identity-service-e2e-tests`. I think that using appId is better, anyways, unless you have strong preferences.

Comment (arsing) [reply docs-dev/e2e-tests.md:45]: No that's fine, I was just curious. Changing it to `.appId` is fine.

You also need to change line 81 in line with lines 78 and 79 (`...` and the comment).

Comment (vadim-kovalyov) [reply docs-dev/e2e-tests.md:49]: Done

File: docs-dev/e2e-tests.md [hunk 3]
```diff
@@ -22,27 +22,31 @@ AZURE_ACCOUNT="$(az account show)"
 AZURE_SUBSCRIPTION_ID="$(<<< "$AZURE_ACCOUNT" jq --raw-output '.id')"
 
 AZURE_RESOURCE_GROUP_NAME='iot-identity-service-e2e-tests'
-AZURE_SP_NAME="http://iot-identity-service-e2e-tests"
+AZURE_SP_NAME="http://iot-identity-service-e2e-tests2"
```
Comment (vadim-kovalyov) [inline docs-dev/e2e-tests.md:25]: opps :( will revert

File: docs-dev/e2e-tests.md [hunk 4]
```diff
@@ -29,20 +29,24 @@ AZURE_LOCATION='...'
 
 az group create --name "$AZURE_RESOURCE_GROUP_NAME" --location "$AZURE_LOCATION"
 
+AZURE_SP=$(az ad sp create-for-rbac --name "$AZURE_SP_NAME" --skip-assignment)
+
 # Save the output of this command. It contains the password for the SP
 # which cannot be obtained later.
-az ad sp create-for-rbac --name "$AZURE_SP_NAME" --skip-assignment
+echo $AZURE_SP
```
Comment (arsing) [inline docs-dev/e2e-tests.md:36]:
```suggestion
echo "$AZURE_SP"
```

**Review by arsing — APPROVED** [review]

**Pull request merged**